templates/iot/examples/infinispan-service.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    description: Hot Rod's port.
+  labels:
+    application: infinispan-server
+  name: infinispan-server-hotrod
+spec:
+  ports:
+  - port: 11222
+    targetPort: 11222
+  selector:
+deploymentConfig: infinispan-server
```

Comment:
I guess there is an indent problem as well

Reply:
yep. it's odd because I tested it and all went fine. fixed